factories/cases.ts

```diff
@@ -57,6 +57,12 @@ export const mockedDeletedCaseNote = caseFactory.build({
   },
   formName: 'adult-case-note',
   deleted: true,
+  deletedFields: {
+    deletedBy: 'jack.musajo@hackney.gov.uk',
+    dateOfDeletion: '2020-12-31',
```

Comment:
Are we definitely returning in this format and not an ISO datetime?

Probably fine if we are but just checking

Reply:
I checked and we are expecting ISO format so I've updated this